server/src/main/java/org/apache/gravitino/server/web/rest/ModelOperations.java

```diff
@@ -309,11 +414,20 @@ public Response getModelVersionByAlias(
   @Produces("application/vnd.gravitino.v1+json")
   @Timed(name = "link-model-version." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
   @ResponseMetered(name = "link-model-version", absolute = true)
+  @AuthorizationExpression(
+      expression =
+          "ANY(OWNER,METALAKE,CATALOG) || "
+              + "((ANY(USE_CATALOG,METALAKE, CATALOG)) "
+              + "&& (SCHEMA::OWNER || (ANY(CREATE_MODEL_VERSION,METALAKE, CATALOG, SCHEMA)) "
```

Comment:
CREATE_MODEL_VERSION could be applied to model, too.

Reply:
fixed